core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -362,10 +362,8 @@ public void moveBlock(long sessionId, long blockId, BlockStoreLocation oldLocati
       }
       return;
     }
-    // TODO(bin): We are probably seeing a rare transient failure, maybe define and throw some
-    // other types of exception to indicate this case.
```

Comment:
This comment is from https://github.com/Alluxio/alluxio/commit/530c4c3d5781a386b962c1cc16d24b3d5b249055 it doesn't make sense here anymore. We have changed how the space allocation works largely. When the move task failed, it can be due to many more reasons than just a transient error here, for example: 
1) The target location is full and this move operation does not evict
2) Eviction failed since the target is full of pinned blocks
3) We evict on the target location but the free space is stolen by someone else
4) maybe an unfound bug from concurrency

Basically this TODO can be removed.